lesson-6/public/index.html

```diff
@@ -5,27 +5,27 @@
   <title></title>
 </head>
 <body>
-  Name:<input type="text" id="a1" />
-  Comment:<input type="text" id="a2" />
-  <input type="submit" id="b1" />
+  Name:<input type="text" id="myName" />
+  Comment:<input type="text" id="myComment" />
+  <input type="submit" id="button" />
 <div>
-  <pre id="c1"/>
+  <pre id="chat"/>
   </div>
   <script>
-const button =document.getElementById("b1");
+const button =document.getElementById("button");
 button.onclick=()=>{
-  const name=document.getElementById("a1").value;
-  const comment=document.getElementById("a2").value;
+  const name=document.getElementById("myName").value;
+  const comment=document.getElementById("myComment").value;
   const xhttp=new XMLHttpRequest();
-  xhttp.open('get', `/XMLHttpRequest?a1=${name}&a2=${comment}`);
+  xhttp.open('get', `/XMLHttpRequest?myName=${name}&myComment=${comment}`);
```

Comment:
/XMLHttpRequest isn't a appropriate url name to handle a request. A url name should be able to reveal what the url does. Plus we already know all back end url handles request